### PR TITLE
add MetaMeute to directory

### PR DESCRIPTION
**Pull request by rxbnDE**

### CODE REVIEW

**Comment by rnestler** [comment]
Looks good according to the validator: https://validator.spaceapi.io/ui/?url=https://status.metameute.de/status/spaceapi.json%22

You could consider implementing v14 of the API. See https://spaceapi.io/ for an example that is compatible with both API versions.

**Review by rnestler — APPROVED** [review]

**Comment by dbrgn** [comment]
@rnestler if you approve, why not merge right away? 🙂

**Comment by rnestler** [comment]
> @rnestler if you approve, why not merge right away? slightly_smiling_face

Well I approved with a suggestion and wanted to give time to react (even if in this case the reaction can also happen after the merge, but habits. Also maybe the chance that people actually look at my response may be higher if it isn't already merged?)

**Pull request merged**
